src/components/Calculator.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Screen from './Screen';
+import Button from './Button';
+
+function Calculator() {
+  return (
+    <div className="w-[9cm] md:w-[12cm] h-[9cm] md:h-[12cm] border m-10 grid grid-cols-4 grid-rows-6">
+      <Screen />
+      <Button value="AC" />
+      <Button value="+/-" />
+      <Button value="%" />
+      <Button value="÷" classextend="bg-orange-500" />
+      <Button value="7" />
+      <Button value="8" />
+      <Button value="9" />
+      <Button value="×" classextend="bg-orange-500" />
+      <Button value="4" />
+      <Button value="5" />
+      <Button value="6" />
+      <Button value="-" classextend="bg-orange-500" />
+      <Button value="1" />
+      <Button value="2" />
+      <Button value="3" />
+      <Button value="+" classextend="bg-orange-500" />
+      <Button value="0" classextend="col-span-2" />
+      <Button value="." />
+      <Button value="=" classextend="bg-orange-500" />
+    </div>
```

Comment:
|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/253159189-ba54ed21-98d4-4b07-ae96-7634ce4f6a0d.png)|![image](https://user-images.githubusercontent.com/99715640/253159224-74ccb80f-c4bb-422c-8e2f-df4ca5067c87.png)|

- [ ] Good job with the job well done 
:clap: but I think we have a design to match here. Kindly apply appropriate styling/colors as seen in the wireframe. It looks like the operands or operation buttons have a different color on them, maybe you should try this: `#de935f`

Reply:
Thank you for the notice! I have resolved that 👏